### PR TITLE
 Show a back to framework application link if framework application info in session

### PR DESCRIPTION
**Pull request by CrystalPea**
Trello ticket: https://trello.com/c/l2xh4cR3/63-confirm-supplier-details-4-return-to-your-framework-application-link

The link and hint text:
screen shot 2018-03-09 at 16 49 03

### CODE REVIEW

**Comment by laurenceberry** [comment]
Looks good to me

File: app/templates/suppliers/details.html [hunk 1]
```diff
@@ -169,4 +169,17 @@
   {% endif %}
 
 {% endcall %}
+
+{% if currently_applying_to %}
+  You must complete your company details to make an application.
```
Comment (Wynndow) [inline app/templates/suppliers/details.html]: Does this line need to be here? I see that it's in the screenshot from the prototype but I'm not sure it's part of this ticket? It feels weird to conditionally display it depending on the framework status. Worth checking with @laurenceberry?

Other than that, this looks good to me!

Comment (constancecerf) [reply app/templates/suppliers/details.html]: This is fine for now but it's a sentence that will change once we build the green confirm button.

**Comment by CrystalPea** [comment]
@Wynndow the content has been confirmed as correct, are you ok to approve or any changes needed? ☺️

**Comment by Wynndow** [comment]
👍

**Pull request merged**